.github/workflows/build.yml

```diff
@@ -43,7 +43,9 @@ jobs:
         run: npm run build -w packages/addons
       - name: Test
         id: 'test'
-        run: npm test -w packages/addons -- --coverage
+        run: |
+          npm run test-check -w packages/addons
+          npm test -w packages/addons -- --coverage
```

Comment:
**nitpick:** split check and test run.
Currently, the upload of the coverage report is done if there are some tsc errors. In this case, the report is not available so the upload will fail.